tool/update_version.dart

```diff
@@ -0,0 +1,156 @@
+import 'dart:convert';
+import 'dart:io';
+
+// This script must be executed from the top level devtools/ directory.
+
+void main(List<String> args) async {
+  final pubspecs = [
+    File('packages/devtools/pubspec.yaml'),
+    File('packages/devtools_app/pubspec.yaml'),
+    File('packages/devtools_server/pubspec.yaml'),
+    File('packages/devtools_shared/pubspec.yaml'),
+    File('packages/devtools_testing/pubspec.yaml'),
+  ];
```

Comment:
nit: create a List<String> and then map it to a List<File>

Reply:
done